### ISSUE TITLE
Generating thumbnail of a certain width - is it possible?

### ISSUE THREAD

**Issue by tomekit** [alann-maulana/image_compression_flutter#11]
In the docs I've found an example:
```dart
Configuration config = Configuration(
  outputType: ImageOutputType.webpThenJpg,
  useJpgPngNativeCompressor: false,
  quality: 40,
);
```

however it doesn't seem there is an option to control the width. 

I have images of random sizes, between xx KBs and couple MBs, having just one quality factor gives too much of a variance to be used for thumbnail generation. It would be great if it's possible to define max width. e.g. 320px.

**Comment by ShahoodulHassan** [comment]
> however it doesn't seem there is an option to control the width.

Have you tried setting `width` and `height` of `ImageFile` that you are providing to `ImageFileConfiguration()`

**Comment by tomekit** [comment]
Not really, since: `ImageFile` within `ImageFileConfiguration` is an input file, not the output configuration.
